.github/workflows/run-build.yml

```diff
@@ -0,0 +1,46 @@
+run the centos image produced by the builder with the repo files mapped in a volume
+runs create.py inside this container
+pushes the results to S3. This thus fully replaces cf.gov-ansible-build
+
+name: 'Run build'
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2.27.0
```

Comment:
Change to `dawidd6/action-download-artifact@v2`? I think I recall using only major versions was the preferred GHA way.